src/murmur/Server.cpp

```diff
@@ -177,7 +177,7 @@ Server::Server(int snum, QObject *p) : QThread(p) {
 			}
 
 			if (::bind(sock, reinterpret_cast< sockaddr * >(&addr), len) == SOCKET_ERROR) {
-				log(QString("Failed to bind UDP Socket to %1").arg(addressToString(ss->serverAddress(), usPort)));
+				log(QString("Failed to bind UDP Socket to %1: %2").arg(addressToString(ss->serverAddress(), usPort), WSAGetLastError()));
```

Comment:
```suggestion
#ifdef Q_OS_WIN
				log(QString("Failed to bind UDP Socket to %1: %2").arg(addressToString(ss->serverAddress(), usPort), WSAGetLastError()));
#else
				log(QString("Failed to bind UDP Socket to %1: %2").arg(addressToString(ss->serverAddress(), usPort), errno));
#endif
```

Reply:
Thanks! I don't have a non-Windows environment to compile this on but I see that's needed due to the CI errors.